src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ZipFileProcessorValidationTest.java

```diff
@@ -24,27 +24,25 @@
 @RunWith(SpringRunner.class)
 public class ZipFileProcessorValidationTest {
 
-    private static final Processor processor = new Processor(
+    private static final EnvelopeProcessor envelopeProcessor = new EnvelopeProcessor(
```

Comment:
class name is `ZipFileProcessorValidationTest`.
Seems like it's actually `EnvelopeProcessor` being tested here (or `Processor` previously).